src/app/feature/calendar/cards/candycane/candycane.component.ts

```diff
@@ -1,13 +1,12 @@
-import {ChangeDetectionStrategy, Component} from '@angular/core';
-import {ChristmasCard} from '../../interfaces/christmas-card';
-import {ChristmasCardType} from '../../enums/christmas-card-type';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ChristmasCardType } from '../../enums/christmas-card-type';
+import type { ChristmasCard } from '../../interfaces/christmas-card';
 
 @Component({
-    selector: 'xmas-candycane',
-    imports: [],
-    templateUrl: './candycane.component.html',
-    styleUrl: './candycane.component.css',
-    changeDetection: ChangeDetectionStrategy.OnPush
+  selector: 'xmas-candycane',
+  templateUrl: './candycane.component.html',
+  styleUrl: './candycane.component.css',
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class CandycaneComponent implements ChristmasCard {
   type = ChristmasCardType.Candycane;
```

Comment:
you can use `readonly`

Reply:
This property seems not to be used, is it safe to remove it (on all components) or did I miss something? @danielglejzner

Reply:
Need to double check - I have no memory on that. Feel free to test it locally with .json file. I might do it today.

Reply:
Yeah I've tested in local and it still works, I will push a commit that remove them.

![image](https://github.com/user-attachments/assets/b4341b0f-cdd3-44b6-a9e3-9af9e326d495)